08-CMSIS/core/include/uart.h

```diff
@@ -1,7 +1,7 @@
 #ifndef __UART_H_
 #define __UART_H_
 
-void usart_init(void);
```

Comment:
Let's move `uart.h` from `core/include` back to `src` directory since it is an implementation-oriented header rather than the interface.